cpp/tensorrt_llm/executor/executorImpl.h

```diff
@@ -79,6 +79,14 @@ class Executor::Impl
     using LlmRequestPtr = std::shared_ptr<batch_manager::LlmRequest>;
     using RequestList = std::list<LlmRequestPtr>;
 
+    struct InTransmissionItem
+    {
+        LlmRequestPtr request;
+        std::optional<SizeType32> lastBlockId; // present when reuse enabled and not variable window
```

Comment:
The logic based on `lastBlockId` in the code context feels a bit obscure. Would it be possible to add a member function or some comments to help with understanding? Thanks!